internal/dag/gatewayapi_processor_test.go

```diff
@@ -26,11 +26,13 @@ import (
 
 func TestComputeHosts(t *testing.T) {
```

Comment:
I think you could change `computeHosts` to not take a full `HTTPRoute`, just the hostnames string slice. It'd make the test setup easier here.

Reply:
👍